src/standalone/Bucket/bucket_parameterizations.jl

```diff
@@ -55,20 +55,26 @@ a helper function which computes and returns the surface evaporative scaling
  factor for the bucket model.
 """
 function ClimaLand.surface_evaporative_scaling(model::BucketModel, Y, p)
-    beta = beta_factor.(Y.bucket.W, Y.bucket.σS, model.parameters.W_f)
+    beta =
+        beta_factor.(
+            Y.bucket.W,
+            Y.bucket.σS,
+            model.parameters.W_f,
+            model.parameters.σS_c,
+        )
     return beta
 end
 
-
 """
     beta_factor(W::FT, σS::FT, W_f::FT) where {FT}
 
-Computes the beta factor which scales the evaporation from the potential
+Computes the beta factor which scales the evaporation/sublimation from the potential
 rate.
 """
-function beta_factor(W::FT, σS::FT, W_f::FT) where {FT}
+function beta_factor(W::FT, σS::FT, W_f::FT, σS_c::FT) where {FT}
     snow_cover_fraction = heaviside(σS)
-    return (1 - snow_cover_fraction) * β(W, W_f) + snow_cover_fraction * 1
+    return (1 - snow_cover_fraction) * β(W, FT(0.75 * W_f)) +
```

Comment:
Do we want to make 0.75 and 0.1 tunable parameters? It would be good to comment on these choices in the doc string.

Reply:
We can make these tunable params - and `p` also?